### ISSUE TITLE
Show more info in "UpdateLiveStats"

### ISSUE THREAD

**Issue by avexus** [PokemonGoF/PokemonGo-Bot#3975]
Can we configure to show more info?

- Such as how many poke/great balls we got.
- When we start up, there is an account info display, like level, xp to next level, how many potions, etc.
Can we include in "UpdateLiveStats" too?

edited by mod

**Comment by icecubeice** [comment]
try validating you config at jsonlint.com
anyway from what I see you have a bracket next to "type" maybe that's what caused it?

**Comment by k4n30** [comment]
@avexus please split this into 2 feature requests so that they can be worked on independantly of each other

**Comment by avexus** [comment]
@icecubeice the config file is fine. I just took the potion that is relevant to this problem.

**Comment by k4n30** [comment]
Waiting on #3950
